app/src/main/java/vn/mbm/phimp/me/Camera2.java

```diff
@@ -651,6 +654,8 @@ public void onPause(){
 	Camera mCamera;
 	private static  final int FOCUS_AREA_SIZE= 300;
 
+	int screen_width = 900,screen_height = 1600;
```

Comment:
Aren't you initializing the variable here? If yes then Pls don't give it a value.